subsys/net/lib/sockets/CMakeLists.txt

```diff
@@ -7,5 +7,6 @@ zephyr_sources(
   sockets_select.c
   )
 zephyr_sources_ifdef(CONFIG_NET_SOCKETS_SOCKOPT_TLS sockets_tls.c)
+zephyr_sources_ifdef(CONFIG_NET_SOCKETS_RAW sockets_raw.c)
```

Comment:
there is an untold "rule": source file are indented to the same column generally (as socket_offload.c below)

Reply:
No idea why its like that.